Documentation/docs/scientific_ecosystem/itk_ecosystem.md

```diff
@@ -0,0 +1,34 @@
+# ITK ecosystem
+
+Together with the core ITK codebase, a number of other projects and
+frameworks are maintained allowing to leverage the power of ITK beyond
+its natural boundaries:
+
+- [ITK/VTK Viewer](https://kitware.github.io/itk-vtk-viewer/docs/index.html)
+  is a web-based image, mesh, and point set viewer.
+- [itk-viewer](https://itk-viewer.on.fleek.co/) is a multi-dimensional
+  web-based image, mesh, and point set viewer.
+- [itk-wasm](https://wasm.itk.org/en/latest/) combines ITK and
+  [WebAssembly](https://webassembly.org/) to enable high-performance
+  spatial analysis in a web browser or system-level environments and
+  reproducible execution across programming languages and hardware
+  architectures.
+- [itkwidgets](https://itkwidgets.readthedocs.io/en/latest/) is an
+  elegant Python interface for visualization on the web platform to
+  interactively generate insights into multidimensional images, point
+  sets, and geometry.
+- [scikit-build](https://scikit-build.org) is a Python build system for
+  CPython C/C++/Fortran/Cython extensions using [CMake](https://cmake.org).
+
+## Legacy tools
```

Comment:
@thewtex Should we remove this as per https://github.com/InsightSoftwareConsortium/ITK/pull/4313#discussion_r1390272297?